doc/man3/OSSL_PARAM_BLD.pod

```diff
@@ -78,8 +73,15 @@ OSSL_PARAM objects of the specified size and correct type for the I<val>
 argument.
 I<val> is stored by value and an expression or auto variable can be used.
 
+When B<I<TYPE>> denotes an integer type, signed integer types will normally
```

Comment:
Is there a case where the param builder won't do this?  I.e. what the _normally_ mean.

Reply:
If you see them mathematically, then yes. BIGNUM is an integer type.

Reply:
In what situation will a signed integer TYPE result in not outputting an integer param?  The only case I'm aware of is BN which is explicitly listed late and is **not** covered by this case.

Two lines down, in what situation will an unsigned integer TYPE result in not outputting an unsigned integer param?  This never happens even for BN.

I.e. remove the two _normally_ words.

Reply:
Uhm, the beginning of DESCRIPTION is generic, and then follow a more detailed description of those functions that are "special" in some way visavi the generic beginning.  Right?  That's at least how I read it.

So, the beginning describes how most integer pushing work, and further down is the description of `OSSL_PARAM_push_BN`, which is "special" visavi all other integer pushing functions.

What did I miss?

Reply:
Even if we accept this (and I'm dubious), the second _normally_ is definitely wrong.  Unsigned integers always map to unsigned integer params.

Reply:
Ah, good point, second *normally* removed